test/core/tsi/alts/crypt/aes_gcm_test.cc

```diff
@@ -803,8 +808,10 @@ static void gsec_test_create_random_aes_gcm_crypter(gsec_aead_crypter** crypter,
                                                     bool rekey) {
   uint8_t* key;
   gsec_test_random_array(&key, key_length);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, nonce_length, tag_length,
-                                   rekey, crypter, nullptr);
+  gsec_aes_gcm_aead_crypter_create(
+      std::make_unique<grpc_core::GsecKey>(absl::MakeConstSpan(key, key_length),
+                                           rekey),
+      nonce_length, tag_length, crypter, nullptr);
```

Comment:
micro-nit: Can you add `/*error_details=*/` in front of `nullptr`?

Reply:
Done.